src/command_modules/azure-cli-netappfiles/azure/cli/command_modules/netappfiles/_params.py

```diff
@@ -0,0 +1,60 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+from azure.cli.core.commands.parameters import tags_type
+
+
+def load_arguments(self, _):
+    with self.argument_context('netappfiles') as c:
+        c.argument('resource_group', options_list=['--resource-group', '-g'], required=True, help='The name of the resource group')
+        c.argument('tags', arg_type=tags_type)
+
+    with self.argument_context('netappfiles') as c:
+        c.argument('account_name', options_list=['--account-name', '-a'], required=True, help='The name of the ANF account')
+
+    with self.argument_context('netappfiles account') as c:
+        c.argument('account_name', id_part='name', options_list=['--account-name', '--name', '-n', '-a'], required=True, help='The name of the ANF account')
+
+    with self.argument_context('netappfiles account list') as c:
+        c.argument('account_name', help='The name of the ANF account', id_part=None)
+
+    with self.argument_context('netappfiles') as c:
+        c.argument('pool_name', options_list=['--pool-name', '-p'], required=True, help='The name of the ANF pool')
+
+    with self.argument_context('netappfiles pool') as c:
+        c.argument('account_name', id_part='name')
+        c.argument('pool_name', id_part='child_name_1', options_list=['--pool-name', '--name', '-n', '-p'], required=True, help='The name of the ANF pool')
+
+    with self.argument_context('netappfiles pool list') as c:
+        c.argument('account_name', options_list=['--account-name', '--name', '-n', '-a'], help='The name of the ANF account', id_part=None)
```

Comment:
There's a lot of duplication of metadata throughout. Not crucial, but you could simplify the maintenance of your module using CLIArgumentTypes. Here's a pattern used in Network.

```Python
account_name_type = CLIArgumentType(options_list=['--account-name', '-a'], id_part='name', help='Name of the ANF account.')

with self.argument_context('netappfiles') as c:
  c.argument('account_name', account_name_type)

with self.argument_context('netappfiles account') as c:
  c.argument('account_name', account_name_type, options_list=['--account-name', '-a', '-n', '--name'])

with self.argument_context('netappfiles pool list') as c:
  c.argument('account_name', account_name_type, id_part=None)
```

This will allow you to change the help text for account name in only one place instead of several, for example.

Reply:
ok, will update this